eyeshade/lib/transaction.js

```diff
@@ -1,27 +1,98 @@
 const BigNumber = require('bignumber.js')
 const getPublisherProps = require('bat-publisher').getPublisherProps
 const uuidv5 = require('uuid/v5')
+const uuid = require('uuid')
 const {
   createdTimestamp,
   normalizeChannel
 } = require('bat-utils/lib/extras-utils')
-
 const getYoutubeChannelId = require('bat-utils/lib/youtube')
 
+const knownChains = {
```

Comment:
currently this doesn't allow for differentiating BAT deposited from an ETH addr and ETH deposited that was converted to BAT

Reply:
do we have information for this? sorry, just not sure what / where we will be getting info from

Reply:
i suppose we would be hitting: https://uphold.com/en/developer/api/documentation/#list-user-transactions